### PR TITLE
Add an example for the heat sink in Icepak.

### PR DESCRIPTION
**Pull request by MaxJPRey**
Add an example for the heat sink in Icepak.
Improve test for the heat sink in Icepak.

Use snake_case in icepak.py
Remove unnecessary 'pass' statement.
Remove new line in icepak tests.

### CODE REVIEW

File: pyaedt/icepak.py [hunk 1]
```diff
@@ -1092,18 +1092,32 @@ def create_parametric_fin_heat_sink(
...
             Tolerance value. The default is ``0.001``.
 
         Returns
         -------
         bool
             ``True`` when successful, ``False`` when failed.
 
+        Examples
+        --------
+
```
Comment (PipKat) [inline pyaedt/icepak.py:1111]: I don't think a blank line is needed here. I'm not sure if it will cause a problem.

Comment (MaxJPRey) [inline pyaedt/icepak.py:1111]:
```suggestion
        --------
```

File: pyaedt/icepak.py [hunk 2]
```diff
@@ -1092,18 +1092,32 @@ def create_parametric_fin_heat_sink(
...
+        Create a symmetric fin heat sink.
```
Comment (MaxJPRey) [inline pyaedt/icepak.py:1112]:
```suggestion
        Examples
        --------
        Create a symmetric fin heat sink.
```

File: pyaedt/icepak.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/icepak.py]: No it's not. good catch.

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**